builtin/settingtypes.txt

```diff
@@ -1680,6 +1680,12 @@ language (Language) enum   ,be,ca,cs,da,de,en,eo,es,et,fr,he,hu,id,it,ja,jbo,ko,
 #    -    verbose
 debug_log_level (Debug log level) enum action ,none,error,warning,action,info,verbose
 
+#    ANSI colored logs: red error log, yellow warning and grey info and verbose logs
+#    "yes" always enables it,
+#    "detect" enables it when printing to terminal (doesn't work on Windows) and
```

Comment:
Wrong line for the Windows notice. It's the entire feature which doesn't work there, not just the `detect` value.

Reply:
fixed